peering/peering.go

```diff
@@ -242,6 +243,22 @@ func (ps *PeeringService) AddPeer(info peer.AddrInfo) {
 	}
 }
 
+// ListPeer lists peers in the peering service.
+func (ps *PeeringService) ListPeer() []peer.AddrInfo {
+	out := make([]peer.AddrInfo, len(ps.peers))
+	c := 0
+	for k, v := range ps.peers {
```

Comment:
nit: use variable names that mean something (id, addrs, etc.).

Reply:
Fixed.